src/pages/SignUpCharity/SignUpCharity.tsx

```diff
@@ -22,6 +27,28 @@ const SignUpCharity: FC = () => {
     setFormData([...removeOldValue, { field, value, section, page }]);
   };
 
+  const { data, isLoading, error } = useQuery({
+    queryKey: ['la'],
+    queryFn: async () => {
+      const { data } = await client.graphql<
+        GraphQLQuery<GetLocalAuthoritiesQuery & GetJoinRequestsQuery>
```

Comment:
Are both the types needed or just GetLocalAuthoritiesQuery

Reply:
thanks, resolved